tests/datasets/plans/test_storage_plan_builder.py

```diff
@@ -116,18 +113,23 @@ def test_storage_query_plan_builder(
     )
     query, snql_anonymized = parse_snql_query(str(snql_query), dataset)
     assert isinstance(query, Query)
-    request = Request(
```

Comment:
I removed this because it was not necessary for the test. Can add it back in if people like